src/applications/disability-benefits/all-claims/containers/ConfirmationPage.jsx

```diff
@@ -29,22 +30,47 @@ export default class ConfirmationPage extends React.Component {
     setTimeout(() => focusElement('va-alert h2'), 100);
   }
 
-  render() {
-    // Reset everything
-    sessionStorage.removeItem(WIZARD_STATUS);
-    sessionStorage.removeItem(FORM_STATUS_BDD);
-    sessionStorage.removeItem(SAVED_SEPARATION_DATE);
-
-    switch (this.props.submissionStatus) {
+  // the legacy 526 confirmation page that has 3 states
+  LegacyConfirmationPage = props => {
+    switch (props.submissionStatus) {
       case submissionStatuses.succeeded:
-        return successfulSubmitContent(this.props);
+        return successfulSubmitContent(props);
       case submissionStatuses.retry:
       case submissionStatuses.exhausted:
       case submissionStatuses.apiFailure:
-        return retryableErrorContent(this.props);
+        return retryableErrorContent(props);
       default:
-        return submitErrorContent(this.props);
+        return submitErrorContent(props);
     }
+  };
+
+  // the new 526 submission confirmation that has one state
+  ConfirmationPageContent = props => (
+    // TODO: #95241 implement new confirmation page
+    <>
+      <Toggler
+        toggleName={Toggler.TOGGLE_NAMES.disability526NewConfirmationPage}
+      >
+        <Toggler.Enabled>
+          <div
+            hidden
+            aria-hidden
+            id="new-confirmation-page"
+            data-testid="new-confirmation-page"
+          />
+        </Toggler.Enabled>
+      </Toggler>
+
+      {this.LegacyConfirmationPage(props)}
```

Comment:
testing was successful! hidden element is not present with toggle off and is present with toggle enabled. appreciate all the unit tests too! are we expecting to wrap the legacy behavior here with `<Toggler.Disabled>` at a later time?

Reply:
> are we expecting to wrap the legacy behavior here with `<Toggler.Disabled>` at a later time?

yes! exactly. we can wrap it or whatever we wanna do. i just wanted to show it was working :) hah